### ISSUE TITLE
Experimental Action Bar file attachment not keyboard accessible

### ISSUE THREAD

**Issue by randall-krauskopf** [primer/view_components#2157]
## What
When the `new_comment_box` feature flag is enabled I've noticed that in my test discussion the file attachment does not respond to space/enter keystrokes when it is focused.

## How

**Note:** I cannot repo this yet on my local, not sure why it would be prod specific as of yet.

1. Navigate to [discussion section](https://github.com/unicorns-r-us/randall-test-repo/discussions/184)
2. Tab into the action bar until focus is present on the "attach files" icon.
3. Press `space` you should observe that the file picker is not triggered.
4. It is clickable though.

https://github.com/primer/view_components/assets/104226843/e1127ac9-3ee0-40e3-9a48-c41b2b294613

**Comment by jonrohan** [comment]
Should be fixed
